ios/Helium/AppDelegate.m

```diff
@@ -48,7 +48,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
                                                    moduleName:@"Helium"
                                             initialProperties:nil];
 
-  rootView.backgroundColor = [[UIColor alloc] initWithRed:1.0f green:1.0f blue:1.0f alpha:1];
+  rootView.backgroundColor = [[UIColor alloc] initWithRed:0.0f green:0.0f blue:0.0f alpha:1];
```

Comment:
good call. I think the primary background color is this
`rootView.backgroundColor = [UIColor colorWithRed: 0.07 green: 0.09 blue: 0.18 alpha: 1.00];`